appveyor.yml

```diff
@@ -14,3 +14,9 @@ build:
 configuration:
 - Debug
 - Release
+test:
+  assemblies:
```

Comment:
This is to ignore HttpMock.dll. Since Nunit 3 has problem to process with HttpMock.dll, I am not sure why,  try to figure out